omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -24,25 +24,28 @@
 		<openmrs:message code="User.needsAPerson"/>
 		<br/>
 		<br/>
-		<table>
-			<tr valign="top">
-				<td style="border-right: 1px lightgrey solid; padding-right: 5em">
-					<h3><openmrs:message code="User.createNewPerson"/></h3>
-					<form method="get" action="user.form">
-						<input type="hidden" name="createNewPerson" value="true"/>
-						<input id="createNewPersonButton" type="submit" value="<openmrs:message code="general.next"/>"/>
-					</form>
-				</td>
-				<td style="padding-left: 5em">
-					<h3><openmrs:message code="User.useExisting"/></h3>
-					<form method="get" action="user.form">
-						<openmrs:message code="User.whichPerson"/> <openmrs_tag:personField formFieldName="person_id" formFieldId="existingPersonId" callback="personSelectedCallback"/>
-						<br/>
-						<input id="useExistingButton" disabled="true" type="submit" value="<openmrs:message code="general.next"/>"/>
-					</form>
-				</td>
-			</tr>
-		</table>
+		<form method="get" action="user.form">
```

Comment:
@rkorytkowski Created one form as you asked and the data in the request is managed quite well :)